### ISSUE TITLE
Atmospheric reflection of lunar halo

### ISSUE THREAD

**Issue by StarFlea** [Stellarium/stellarium#3478]
![Halo](https://github.com/Stellarium/stellarium/assets/118836358/e957b04a-7ddf-485f-8dcc-c8cc255a791f)

Hello! The screenshot shows the problem that occurs. It shows up when the ground is switched on or off:
The lunar halo is reflected below the horizon. Apparently only the visual atmosphere model “Preetham” is affected, the effect is not visible on “ShowMySky”.

Best regards!
StarFlea

### System
* Stellarium version: 23.3+ (23.3.296-70e4f4d) Qt5
* Operating system: Windows 10
* Graphics Card: NVIDIA GeForce GTX 1660 Ti (with newest driver)
* Screen type (if applicable): 1920x1080

**Comment by gzotti** [comment]
Show proof it is not there... :-)

**Comment by 10110111** [comment]
> Show proof it is not there...

It was removed by efa6b054486f5b0095103ff39131ee82f73c8c70.

But the presence of this effect in Preetham model took me by surprise. Has it always been there?

**Comment by gzotti** [comment]
Yes, I think so. Given that using atmosphere should also always use landscape, it was nothing to be worried. The original model AFAICR has  no sub-horizon information, and the original implementation therefore just mirrored the sky. 
There is also no known easy improvement for the refraction discontinuity in a zone unobservable from ground-based locations on earth. Nobody cares about that.

**Comment by gzotti** [comment]
I also had in mind you had removed that, but why does OP report it for 23.3+?
And BTW, the Preetham model has no moon glare at all.

**Comment by 10110111** [comment]
> I also had in mind you had removed that, but why does OP report it for 23.3+?

I removed it for ShowMySky. As far as I'm concerned, Preetham is in maintenance mode.

> And BTW, the Preetham model has no moon glare at all.

It's not glare, it's aureole (i.e. you can photograph it even with the moon hidden behind an obstacle), and Preetham does show it since at least 2014.

**Comment by gzotti** [comment]
Right, aureole. I just did not have the word in my mind. It's also not halo. 
I meant, the original model by Preetham does not have it.  Our implementation is a mixed Preetham/Schaefer model from before 2010. 
Wow, moonrise! Getting ready for the eclipse...

**Comment by 10110111** [comment]
> Wow, moonrise! Getting ready for the eclipse

Oh, thanks for notifying :)

**Comment by StarFlea** [comment]
What has changed in the atmosphere model? The phenomenon still appears in the new version 25.0.77-e4ca6a7...

**Comment by gzotti** [comment]
Nothing. And? Imagine standing in a field wet from dew.

**Comment by StarFlea** [comment]
...without ground? ;-)

No, seriously, the current release notes mentioned changes to the atmosphere model, so I checked to see if the phenomenon still exists...

**Comment by gzotti** [comment]
Right. Do something impossible in nature like switching off the ground to discover something probably implausible like a mirror artifact within a graphic model in a place usually covered. But as asked previously: Prove that it's wrong. This mirror effect is harmless because it is covered by ground in realistic simulation scenarios. The recent changes did not concern your concern, else it would have been flagged.
